app/views/carts/checkout.html.haml

```diff
@@ -1,11 +1,19 @@
-Hey, you're checking out.
-
-You ordered:
-%br
-  - @order.shopping_cart_items.each do |dish|
-    = dish.item.dish_name
-Total price:
-= @order.total.to_i
-%p
-  Delivery address:
-  = current_user.address
+.h1-top
+  %h1 = "Hey #{current_user.name}, you're checking out"
```

Comment:
Is this view displayed AFTER processing of payment? If so we probably want to say "You have checked out" or something...